stdlib/public/core/MemoryLayout.swift

```diff
@@ -40,10 +40,7 @@
 ///             byteCount: count * MemoryLayout<Point>.stride,
 ///             alignment: MemoryLayout<Point>.alignment)
 @frozen // namespace
-public enum MemoryLayout<T: ~Copyable>: Copyable {}
-
-@available(*, unavailable)
-extension MemoryLayout: _BitwiseCopyable {}
+public enum MemoryLayout<T: ~Copyable>: ~_BitwiseCopyable, Copyable {}
```

Comment:
Why do we care that MemoryLayout is not bitwise copyable? It has no cases anyway.

Reply:
That was requested here: https://github.com/apple/swift-evolution/pull/2314#discussion_r1508201954 .